src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py

```diff
@@ -904,7 +1083,10 @@ def test_import_zaaktype_besluittype_already_exists(self, *mocks):
         )
 
         response = form.submit("_import_zaaktype").follow()
-        response = response.form.submit("_select")
+
+        form = response.form
+        form["besluittype-0-existing"].value = ""
+        response = form.submit("_select")
```

Comment:
Why have this test and the test below been changed?

Reply:
It now automatically selects the correct option when it should test what happens if you select the wrong option.

Reply:
Thank for explanation!